tests/tools/src/lib.rs

```diff
@@ -76,6 +76,10 @@ pub fn scripted_fixture_repo_read_only_with_args(
         .to_owned();
     let script_result_directory = fixture_path(Path::new("generated").join(format!("{}", script_identity)));
     if !script_result_directory.is_dir() {
+        if script_result_directory.exists() {
```

Comment:
Some tests directories take a long time to create, sometimes up to a minute. What about renaming `"generated"` into `"generated-do-not-edit"`?